assessments/projects/rook/self-assessment.md

```diff
@@ -0,0 +1,374 @@
+
+## Project compliance
+
+There are no security standards or sub-sections the project is already documented as meeting.
+
+## Secure development practices
+
+#### Development Pipeline.  
+Rook's development pipeline is designed to ensure that the software is robust, reliable, and secure. It involves several stages of testing and assessment as the software is developed and built.
+
+#### Contributor Requirements
+
+Contributors to Rook are required to sign their commits, adhering to the Developer Certificate of Origin (DCO). This practice ensures the integrity of the code by verifying that the changes are made by the person who claims to have made them. Contributors use the Signed-off-by line in commit messages to signify their adherence to these requirements. Git has a -s command-line option to append this automatically to commit messages
+Rook leverages a DCO bot to enforce the DCO on each pull request and branch commits. This bot helps ensure that all contributions are properly signed off.
+Contributors can get started by forking the repository on GitHub, reading the installation document for build and test instructions, and playing with the project 
+
+#### Container Images
+
+The container images used in Rook are immutable, which means they cannot be changed after they are created. This practice enhances the security of the software by preventing unauthorized modifications.
+
+#### Reviewers
+
+Before a commit is merged, it is reviewed by multiple reviewers. This practice helps catch potential security issues early in the development process. The exact number of reviewers required before merging is not specified in the documentation.
+Rook empowers contributors to approve and merge code changes autonomously. The maintainer team does not have sufficient resources to fully review and approve all proposed code changes, so trusted members of the community are given these abilities. The goal of this process is to increase the code velocity of all storage providers and streamline their day-to-day operations, such as pull request approval and merging
+
+#### Automated Checks
+
+Rook includes automated checks for vulnerabilities. These checks are part of Rook's continuous integration (CI) process and automatically run against every pull request. The results of these tests, along with code reviews and other criteria, determine whether a pull request will be accepted into the Rook repository
+
+#### Integration Tests
+
+Rook's upstream continuous integration (CI) tests will run integration tests against your changes automatically. You do not need to run these tests locally, but you may if you like.
+
+
+* Communication Channels.
```

Comment:
This statement might need to be formatted to be a heading

Reply:
Fixed in this quick commit (I'll be happy to squash these later, just want to keep the commits while we're still editing the project): [3774c00](https://github.com/dhauss/tag-security/commit/3774c006f129416e211a396c2858f0f774307b1e)